api/Controllers/AccountsController.cs

```diff
@@ -56,7 +65,47 @@ public async Task<IActionResult> GetAccountEmail([FromRoute] int Id)
             return Ok(email);
         }
 
+        [Route("{id}")]
+        [HttpPut]
+        [Authorize]
+        public async Task<IActionResult> ChangeAccountPassword([FromBody] ChangePassword request)
+        {
+            if (request == null)
+            {
+                return BadRequest("Not enough data");
+            }
+
+            int id = Convert.ToInt32(User.Claims.Where(claim => claim.Type == "sub").First().Value);
```

Comment:
Have replaced my code to code which uses User.Identity.Name. Do not like the way my token looks right now because of long key for name. Gonna investigate this moment and try to use shorter key for name value